subprojects/ivy/src/test/groovy/org/gradle/api/publish/ivy/tasks/PublishToIvyRepositoryTest.groovy

```diff
@@ -63,7 +67,10 @@ class PublishToIvyRepositoryTest extends AbstractProjectBuilderSpec {
         publish.publication = publication
 
         then:
-        publish.inputs.files.files == publishableFiles.files
+        GetInputFilesVisitor visitor = new GetInputFilesVisitor(publish.toString())
+        def walker = new DefaultPropertyWalker(new DefaultPropertyMetadataStore([]))
+        TaskPropertyUtils.visitProperties(walker, publish, visitor)
+        visitor.files.files == publishableFiles.files
```

Comment:
Shall we move this to `TaskPropertyTestUtils`?

Reply:
moved.